### PR TITLE
[2/2] Wigner gif

### PR DESCRIPTION
**Pull request by abocquet**
And fixes a bug with exp ops that appeared with JAX upgrade I think
Part of 
- https://github.com/dynamiqs/dynamiqs/pull/416
- [2/2]

### CODE REVIEW

File: dynamiqs/plots/plots_wigner.py [hunk 1]
```diff
@@ -224,3 +230,64 @@ def plot_wigner_mosaic(
             clear=False,
         )
         ax.set(xlabel='', ylabel='', xticks=[], yticks=[])
+
+
+def plot_wigner_gif(
+    states: ArrayLike,
+    length_s: int = 5,
```
Comment (gautierronan) [inline dynamiqs/plots/plots_wigner.py:237]: What is this? Could we find a more explicit name in the absence of documentation?

Comment (abocquet) [reply dynamiqs/plots/plots_wigner.py:237]: It plots a wigner in a gif. I will write a doc

Comment (gautierronan) [reply dynamiqs/plots/plots_wigner.py:237]: How about `gif_duration` then?

Comment (gautierronan) [reply dynamiqs/plots/plots_wigner.py:237]: And it should be a float I guess.

**Review by gautierronan — APPROVED** [review]

**Pull request merged**